tests/_utils/wait_process_exit

```diff
@@ -1,29 +1,16 @@
 #!/bin/bash
 # parameter 1: process name
-process=$1
-
-PID=$(pgrep $process)
-if [ -z "$PID" ]; then
-	echo "process $process already exit"
-	exit
-fi
 
-WAIT_COUNT=10
-KILL_TIMES=1 # make sure only send kill -9 once.
-count=0
+process=$1
 
-while true; do
-	pgrep $process >/dev/null 2>&1
-	ret=$?
-	if [ "$ret" != "0" ]; then
-		echo "process $process already exit"
-		break
-	fi
-	sleep 0.2
-	((count++))
-	echo "wait process $process exit..."
-	if [ $count -gt $WAIT_COUNT ] && [ $KILL_TIMES -lt 2 ]; then
-		((KILL_TIMES++))
-		kill -9 $PID || true
-	fi
+while true
```

Comment:
What about only wait finite times and returns failure if exceed to avoid infinite loop if there is a dead loop in target program 🤔

Reply:
https://github.com/pingcap/dm/pull/1773/commits/90fa90e3d456345d4a782c0d50b450a3590af980